paasta_tools/deploy_chronos_jobs

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 if am_i_mesos_leader >/dev/null; then
-  list_chronos_jobs | shuf | xargs -n 1 -r -P 5 setup_chronos_job
+  list_chronos_jobs | tr ' ' '\n' | shuf | xargs -n 1 -r -P 5 setup_chronos_job
```

Comment:
I thought about that, but I got worried that people might have scripts parsing that output (similar to how we do that with deploy_chronos_jobs) which would break after such a change. This approach seemed like the easiest and safest solution to the problem.

If we are fine possibly breaking some scripts, I can modify list_chronos_jobs.